docs/operate/get-started/other-hardware/manage-modules.md

```diff
@@ -90,10 +90,11 @@ At the end of your <file>meta.json</file>, add the build configuration:
   "module_id": "example-module",
   ...
   "build": {
-    "setup": "./setup.sh", // optional - command to install your build dependencies
-    "build": "./build.sh", // command that will build your module
-    "path" : "dist/archive.tar.gz", // optional - path to your built module
-    "arch" : ["linux/amd64", "linux/arm64"] // architecture(s) to build for
+    "setup": "./setup.sh", // optional - command for one-time setup
```

Comment:
Could this be more descriptive? I like how the next one is moreso that before

Reply:
you're welcome to make it more descriptive but that goes beyond the quick fix I am making here